_meta/beat.yml

```diff
@@ -4,13 +4,13 @@
 
 apm-server:
   # Defines the host and port the server is listening on
-  host: "localhost:8080"
 
-  #server.max_unzipped_size:
-  #server.max_header_bytes:
-  #server.read_timeout: 2s
-  #server.write_timeout: 2s
+  #host: "localhost:8080"
```

Comment:
@simitt @roncohen Could we please have your thoughts on this change config change? We will keep it like this for probably quite a while.